server/scsynth/SC_CoreAudio.cpp

```diff
@@ -536,11 +536,11 @@ SC_CoreAudioDriver::~SC_CoreAudioDriver()
 }
 
 std::vector<AudioValueRange> GetAvailableNominalSampleRates(const AudioDeviceID& device) {
-	std::vector<AudioValueRange>	result;
-	OSStatus						err;
-	UInt32							size;
-	UInt32							count;
-	std::auto_ptr<AudioValueRange>	validSampleRateRanges;
+	std::vector<AudioValueRange>		result;
+	OSStatus							err;
+	UInt32								size;
+	UInt32								count;
+	std::unique_ptr<AudioValueRange[]>	validSampleRateRanges;
```

Comment:
I tried to keep the existing whitespace conventions, but in general I'm more a fan of spaces and keeping types close to variables instead of these tabulated layouts. 

Was it ever contemplated to use a tool like clang-format (maybe as part of a pre-commit hook in git) to ensure consistent formatting?

Reply:
> in general I'm more a fan of spaces and keeping types close to variables instead of these tabulated layouts.

Same

> Was it ever contemplated to use a tool like clang-format (maybe as part of a pre-commit hook in git) to ensure consistent formatting?

Yes, we'd like to get linting or stable formatting going, but right now the codebase is kind of a nightmare in that regard (mix of 2-spaces, 4-spaces, tabs, and alignment styles). To be fixed soon, hopefully.